docs/pages/guides/mesheryctl/configuring-autocompletion-for-mesheryctl.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+title: Configuring Autocompletion for `mesheryctl`
+permalink: guides/mesheryctl/configuring-autocompletion-for-mesheryctl
+language: en
+type: Guides
+category: mesheryctl
+---
+# Configuring Autocompletion for `mesheryctl`
+
+If you would like to have `mesheryctl` commands automatically completed for use as you use `mesheryctl`, then use the following instructions to configure automatic completion within your environment.
+
```

Comment:
For this line alone, the page prints like this....

Reply:
![image](https://user-images.githubusercontent.com/59508546/113428313-3545c400-93f4-11eb-802c-5cda75a4dfb6.png)

Reply:
Sounds like the frontmatter on this page could use: `display-title: "false"`.

Reply:
Yeah, need to look on that